src/components/RightPane/SectionTable/PrereqTree.tsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable prefer-const */
```

Comment:
Why do we need the eslint disable here? Which lines offend the linter and why can't they be fixed to adhere to the rule? It'd be better to disable the linter checks per-line instead of per-file.

Reply:
It's because "let" was used inside the function components. They should all be safely converted to "const". 

A fun part of react is how const isn't const because the entire function is re-run per state update. So "let" and "const" are effectively the same unless you explicitly re-assign halfway through the function.